hpedockerplugin/hpe_storage_api.py

```diff
@@ -228,6 +239,9 @@ def volumedriver_create(self, name, opts=None):
                                               "for mountConflictDelay. Please"
                                               "specify an integer value." %
                                               mount_conflict_delay_str})
+            if ('backend' in contents['Opts'] and
+                    contents['Opts']['backend'] != ""):
+                current_backend = str(contents['Opts']['backend'])
```

Comment:
Will it be better to check whether passed backend is present in orchestrator._manager ?
Throw an error if backend is not present in hpe.conf